client/plots/profilePlot.js

```diff
@@ -624,6 +623,10 @@ export async function loadFilterTerms(config, app, opts) {
 	config.filterTWs = twlst
```

Comment:
Syntax improvement: in `twlst.push()` in line 609 or 608, no need to create an array and then spread it again, should just be 
```js
twlst.push(
	config.countryTW,
	config.regionTW,
	config.incomeTW,
	config.typeTW,
	config.teachingStatusTW,
	config.referralStatusTW,
	config.fundingSourceTW,
	config.hospitalVolumeTW,
	config.yearOfImplementationTW
)
```

Reply:
thanks done